.github/workflows/ci.yml

```diff
@@ -106,27 +113,43 @@ jobs:
 
   try-runtime:
     runs-on: ubuntu-latest-16-cores
-    timeout-minutes: 45
+    timeout-minutes: 50
     steps:
       - uses: actions/checkout@v3
-
-      - name: Install latest stable Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: stable
-          target: ${{ env.target }}
+      - name: Check Version
+        run: |
+         echo -n "eden_rev=" >> $GITHUB_OUTPUT
+         curl -s --request POST   --url https://nodle-parachain.api.onfinality.io/public   --header 'Content-Type: application/json'   --data '{
```

Comment:
```suggestion
         curl -s --request POST  --url ${{ env.try-runtime-uri }} --header 'Content-Type: application/json'  --data '{
```

Reply:
They are not the same:  `try-runtime-uri: wss://eden-rpc.dwellir.com:443` one is https and the other is wss (which is https)

Reply:
Ah ok I missed to see. Would it be good to define this uri in env as well?